app/activity/[title]/page.tsx

```diff
@@ -0,0 +1,96 @@
+'use client'
+import {
+  Heading,
+  Box,
+  Container,
+  useColorModeValue,
+  Flex,
+  Spacer,
+  Text,
+  Textarea,
+  Input,
+  FormControl,
+  FormLabel
+} from '@chakra-ui/react'
+import { FiChevronLeft, FiClock, FiMapPin, FiLink2 } from 'react-icons/fi'
+import { Carousel } from '@/components/carousel'
+import { Link } from '@/components/link'
+import { Header, Footer } from '@/components/navigation'
+import { TrashIcon, EditIcon } from '@/icons'
+import { customColors } from '@/theme/color'
+
+export default function ActivityDetails({ params }: { params: { title: string } }) {
+  const bg = useColorModeValue('white', 'gray.800')
+  const color = useColorModeValue('black', 'gray.300')
+  return (
+    <>
+      <Header />
+      <Box as="main" minH="100vh" bg={bg} color={color}>
+        <Container
+          maxW={{ base: '710px' }}
```

Comment:
Without padding, max width should be 710px. Could you use the container valuable 'container.md'?  But you also need to change to '742px' considering the padding. Sorry 🙏 
Please check app/theme/index.ts file :)